### DISCUSSION TITLE
Not able to get a password

### DISCUSSION THREAD

**Discussion by GunnarBjoru** [Chouffy/home_assistant_tgtg#9] [Q&A] [answered]
Hi, I'm not able to reset my password. When I log on I'm just sent a email with a link and a 6 digit number.
I can use both, but none of them gives me the opportunity to set a password. Any ideas?

**Comment by Chouffy** [comment]
@GunnarBjoru Another user experienced [this issue](https://github.com/Chouffy/home_assistant_tgtg/issues/2#issue-1053485267) and it says that you have a reset password link in the "login mail" that TGTG sends when a new device is connected. Could you check this?

**Comment by Chouffy** [comment, marked as answer]
Maybe try here: https://store.toogoodtogo.com/reset-password

**Reply by GunnarBjoru** [reply]
This link made me able to reset the password. Thank you very much.

**Comment by AngryWhopper** [comment]
Hi, the reset-password does not seem to work anymore. Does anyone have another way to set a password? The tgtg_get_token.py now says Check your mailbox on PC to continue and then times out. The e-mail I get has no way of setting a password.

**Reply by AngryWhopper** [reply]
And the link above doesn't send me a e-mail at all

**Reply by Chouffy** [reply]
@AngryWhopper TooGoodToGo doesn't use passwords anymore, and instead uses links to validate authentication. So you have to:
1. Keep your terminal open at the "Check your mailbox" step
2. Go to your mail client on PC (not mobile), and click on the link
3. Back to the terminal, wait a bit, and voilà! you should be logged in and be able to get the tokens

**Reply by AngryWhopper** [reply]
Thank you for your quick response. My mail client did not recognize the link in the e-mail so the mail was empty. In the sourcecode of the e-mail I could get the link and paste it in the browser. That worked perfectly!
